src/header/common/access_control/allow_headers.rs

```diff
@@ -0,0 +1,25 @@
+use std::fmt::{mod};
+
+use header;
+use header::shared;
+
+#[deriving(Clone)]
+struct AccessControlAllowHeaders(pub Vec<String>);
```

Comment:
Probably it should be there.
But I do not exactly know what the `deref!` is doing and for what we need it.